packages/generator-common/package.json

```diff
@@ -37,15 +37,13 @@
     "readme": "ts-node ../../scripts/replace-common-readme.ts"
   },
   "dependencies": {
-    "@sap-cloud-sdk/util": "^2.8.0",
```

Comment:
[req] Perhaps this is not intended after the release?

Reply:
You're right, I reverted it.